demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1103,6 +1103,45 @@ def test_is_there_spaces_in_the_end_of_name(self, pack: Pack, answer, dashboard_
         res_validator = IntegrationValidator(structure)
         assert res_validator.is_there_spaces_in_the_end_of_name() is answer
 
+    def test_ignore_test_doc_non_pack_file_should_not_ignore(self):
+        """
+        Given
+            - File path
+        When
+            - File should not be ignored
+        Then
+            - File is not ignored and False is returned
+        """
+        file_path = 'Packs/SomeIntegration/IntegrationName/file.py'
+        validate_manager = ValidateManager(check_is_unskipped=False)
+        assert not validate_manager.ignore_test_doc_non_pack_file(file_path)
+
+    def test_ignore_test_doc_non_pack_file_test_file(self):
+        """
+        Given
+            - File path
+        When
+            - File is part of the test_data directory
+        Then
+            - File is ignored and True is returned
+        """
+        file_path = 'Packs/SomeIntegration/IntegrationName/test_data/file.json'
```

Comment:
this is not a valid path for test data.
test data should appear under one of the following:
* `Packs/pack_id/test_data`
* `Packs/pack_id/Integrations/integration_id/test_data`
* `Packs/pack_id/Scripts/script_id/test_data`
* `Packs/pack_id/TestPlaybooks/test_data`

we can consider allowing freestyle location for the folder but it feels messy...

Reply:
fixed